### DISCUSSION TITLE
Useful Metabarcoding resources

### DISCUSSION THREAD

**Discussion by skreling** [sr320/course-fish546-2021#32] [Show and tell]
http://sixthresearcher.com/wp-content/uploads/2017/03/metabarcoding_workshop_theory.pdf

Wolf scat metabarcoding Tutorial
https://pythonhosted.org/OBITools/wolves.html

https://replikation.github.io/bioinformatics_side/metagenome/metabarcoding/

**Comment by sr320** [comment]
Thanks! but 2 of the links do not work.. Note when using Markdown you need to include the URL itself, but in GitHub you can also just include the URL as text and GH will automatically hyperlink

**Comment by jdduprey** [comment]
Thanks Samantha, these are great intro to metabarcoding resources!
